### PR TITLE
Tidying up DDL scripts, combining groundzero scripts and removing unnecessary scripts

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
Updated groundzero scripts to match current prod environment

# What has changed
Updated main groundzero scripts and removed unnecessary scripts from repository 

# How to test?
Wipe database/schemas and recreate in dosql-i-know-what-i-am-doing and add:

- name: SPRING_JPA_HIBERNATE_DDL_AUTO
   value: validate 

...to bottom of census-rm-kubernetes - action-scheduler-deployment.yml/uac-qid-service-deployment.yml/case-processor-deployment.yml then redeploy these.

From census-rm-kubernetes/optional, redeploy notify-stub-deployment.yml/notify-stub-service.yml so that these appear in list of pods in GCP environment then run acceptance tests

# Links
https://trello.com/c/nQOfMX9w/353-update-groundzero-script-to-create-database-schema8

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[Update groundzero script to create database schema.(8)](https://trello.com/c/nQOfMX9w/353-update-groundzero-script-to-create-database-schema8)

File: groundzero_ddl/actionv2.sql [hunk 1]
```diff
@@ -1,66 +1,96 @@
...
+
+CREATE INDEX case_id_idx ON actionv2.cases (case_id);
+
+CREATE INDEX lsoa_idx ON actionv2.cases (lsoa);
+
+CREATE INDEX receipt_received_idx ON actionv2.cases (receipt_received);
+
+CREATE INDEX treatment_code_idx ON actionv2.cases (treatment_code);
+
+CREATE INDEX case_id_uac_qid_link_idx ON uac_qid_link (case_id);
```
Comment (gavinleeedwards) [inline groundzero_ddl/actionv2.sql]: Can you add the actionv2 schema name to the table name.

Comment (LeoKHoward) [reply groundzero_ddl/actionv2.sql]: Done

File: groundzero_ddl/casev2.sql [hunk 2]
```diff
@@ -1,65 +1,84 @@
...
-    alter table if exists event 
-       add constraint FKamu77co5m9upj2b3c1oun21er 
-       foreign key (uac_qid_link_id) 
-       references uac_qid_link;
 
-    alter table if exists uac_qid_link 
-       add constraint FKgqdn6408vhuo465xp8tlibp7c 
-       foreign key (caze_case_ref) 
-       references cases;
+CREATE INDEX case_id_idx ON casev2.cases (case_id);
```
Comment (liamtoozer) [inline groundzero_ddl/casev2.sql]: Think `case_id_idx` can be removed from here and replaced with the unique constraint/index for the case and action `cases` tables [here](https://github.com/ONSdigital/census-rm-ddl/blob/master/manual_scripts/schema_and_data_migration/0013-add-unique-constraint-case-id.sql#L17-L21)?

**Pull request merged**